app/services/vine_voucher_redeemer_service.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class VineVoucherRedeemerService
+  attr_reader :order, :errors
+
+  def initialize(order: )
+    @order = order
+    @errors = {}
+  end
+
+  def call
+    # Do nothing if we don't have a vine voucher added to the order
+    voucher_adjustment = order.voucher_adjustments.first
```

Comment:
`Order` `has_many :voucher_adjustments`, but "[We only support one voucher per order for now](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/services/voucher_adjustments_service.rb#L42)".

As we continue to develop voucher functionality, more `first`s like this appear in the code, and we have to just "know" that we only support one for now.. But let's live with that for a bit longer.